scripts/install.sh

```diff
@@ -14,12 +14,21 @@ echo "Starting installation."
 
 # Determine release filename. This can be expanded with CPU arch in the future.
 if [ "$(uname)" == "Linux" ]; then
-	OS="Linux"
+    OS="Linux"
 elif [ "$(uname)" == "Darwin" ]; then
-	OS="Darwin"
+    OS="Darwin"
 else
-	echo "This operating system is not supported."
-	exit 1
+    echo "This operating system is not supported. The supported operating systems are Linux and Darwin"
+    exit 1
+fi
+
+if [ "$(uname -m)" == "x86_64" ]; then
+    MACHINE="x86_64"
+elif [ "$(uname)" == "aarch64" ]; then
+    MACHINE="aarch64"
+else
+    echo "This machine architecture is not supported. The supported architectures are x86_64 and aarch64."
+    exit 1
```

Comment:
Just curious - we're building for arm 6/7, but not looking for that machine type here (or at least I don't think we are). Should this be adjusted will this work and I'm just missing something?

Reply:
Yeah, I want to see what the archive names are and then I can update the script.

Reply:
Or should we just not build for 32-bit architectures?

Reply:
But that will take a release.

Reply:
Cool, makes sense